backend/tests/Squidex.Web.Tests/Squidex.Web.Tests.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="FakeItEasy" Version="5.4.0" />
     <PackageReference Include="IdentityServer4" Version="3.0.2" />
     <PackageReference Include="IdentityServer4.AspNetIdentity" Version="3.0.2" />
+    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
```

Comment:
Whats that?

Reply:
Reverted. There were issues while building "Squidex.Web.Tests", "Squidex.Infrastructure.Tests" and "Squidex.Domain.Users.Tests" projects, so I had to add these dependencies, but now it works without them as well. Probably I had to delete the NuGetFallbackFolder after installing 3.0 SDK, that might have caused this issue.